docs/tutorial/exceptions.rst

```diff
@@ -0,0 +1,54 @@
+Handling exceptions
+===================
+
+.. highlight:: python
+
+All exceptions are located in the ``fedora_messaging.exceptions`` module.
+
+When publishing
```

Comment:
It'd be good if all the information here was neatly included in the API documentation. I see I didn't document that ``publish`` raises a ValidationError, so if we included that and more detailed doc-blocks on the exception classes, you should be able to just point to the ``fedora_messaging.api.publish`` API docs.

Reply:
I fixed the rest but I'm not entirely sure how you picture that part. I've added the references and enhanced the doc-blocks. Do you mean this entire section should be replaced by a reference to the API docs?

Reply:
Just more helpful error handling documentation for the publish API. I can shuffle things around in a follow-up PR so feel free to merge this.